src/test/java/net/openhft/chronicle/bytes/MappedMemoryTest.java

```diff
@@ -134,8 +135,11 @@ public void withRawNativeBytesTess()
                     }
                     bytes.releaseLast(test);
 
+                    Jvm.perf().on(getClass(), "With NativeBytes,\t\t time= " + 80 * (System.nanoTime() - startTime) / BLOCK_SIZE / 10.0 + " ns, number of longs written=" + BLOCK_SIZE / 8);
+                } catch (Throwable throwable) {
+                    // Performance test so just make sure the test ran
+                    fail(throwable.getMessage());
```

Comment:
why not use assertDoesNotThrow was well ?

Reply:
It will take some time to fix so I left it as is.